### PR TITLE
Create a example on how to fix PocketMine's broken recipe ids

### PR DESCRIPTION
**Pull request by AnyBananaGAME**

### CODE REVIEW

**Comment by extremeheat** [comment]
This is not a fix

**Comment by AnyBananaGAME** [comment]
> This is not a fix

It fixes the issue with it, it's supposed to be a temporary fix

**Comment by extremeheat** [comment]
If you want to change recipe names in your code you can go ahead and do that, but this does not fix anything. A fix implies correcting an error, not replacing some data for unknown reasons.

File: examples/PocketMineRelay.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+relay.listen() // Tell the server to start listening.
+
+relay.on('connect', player => {
+  console.log('New connection', player.connection.address)
+
+  // Server is sending a message to the client.
+  player.on('clientbound', ({ name, params }) => {
+    if (name === 'crafting_data') {
+      // The RecipeIDs that are sent by PocketMine are not being recognised and
+      // the relay thinks its an empty string (If you try to console.log it that will log nothing)
```
Comment (extremeheat) [inline examples/PocketMineRelay.js:39]: JavaScript does not use null terminated strings, so whether the string has a \0 or not in it somewhere will make no difference. Length of the string is stored separately in memory. If this fixes the server/client from crashing (or recipes work now), then please make that clear and we can reopen the PR.

**Pull request closed without merging**